src/main/java/org/jvnet/hudson/annotation_indexer/AnnotationProcessorImpl.java

```diff
@@ -80,6 +81,10 @@ void add(Element elt) {
             case CONSTRUCTOR:
                 t = (TypeElement) elt.getEnclosingElement();
                 break;
+            case PACKAGE:
+                classes.add(((PackageElement)elt).getQualifiedName().toString());
```

Comment:
This is not a class name. That will break [here](https://github.com/jenkinsci/lib-annotation-indexer/blob/1da4d7c9a800fd1beb61895355e19f421a4c4ab3/src/main/java/org/jvnet/hudson/annotation_indexer/Index.java#L99-L107).

Reply:
indeed. And java reflexion don't let us list classes in a package using reflexion :'(
What do you think about the [initial commit](https://github.com/jenkinsci/lib-annotation-indexer/pull/5/commits/f913f1a43a8668d1fca3f795df0fdd70d043f7f8) to just add all classes within package to the index ? this makes the accmod implementation less efficient, but would reduce impact ?

Reply:
updated by https://github.com/jenkinsci/lib-annotation-indexer/pull/5/commits/1952a83f42722e94ad4e7d5dcc993f1d6331628e to index com.foo.* and list Package as indexed element

https://github.com/kohsuke/access-modifier/pull/13 updated accrodingly